src/Service/CaseManager.php

```diff
@@ -99,4 +103,47 @@ public function newCase(CaseEntity $caseEntity, Board $board): CaseEntity
 
         return $caseEntity;
     }
+
+    public function updateDeadlineBooleans($isDryRun)
+    {
+        // We do this in batches as there might be a large number of cases
+        // @see https://www.doctrine-project.org/projects/doctrine-orm/en/2.9/reference/batch-processing.html#iterating-results
+        $batchSize = 20;
+        $i = 1;
```

Comment:
I don't know if it's only me, but I think it would be better with a more descriptive name for the counter variable e.g. `$caseCounter` or something similar.

Reply:
Agreed, renamed to `$caseCounter`.